LabManagement/PowerShell/BulkOperations/Az.LabServices.BulkOperations.psm1

```diff
@@ -1438,6 +1438,178 @@ function Remove-AzLabUsersBulk {
     }
 }
 
+# This function is used to send lab invitations *after* a lab has been created\published - for example, several days\weeks later when student enrollment is finalized.
```

Comment:
Just to confirm - when the school uses the New-AzLabsBulk method, we automatically send the user invitations...  Should we remove the code from there?  Or we expect that the customer would know NOT to include student emails in the original CSV file when creating labs and add them later?

Reply:
Yes, the school in this case does explicitly comment out the code in the section of New-AzLabsBulk so that the invitations are sent during lab creation.   When they create the labs, they also *don't* add the student emails in the csv.

Reply:
ok, sounds good!